numberformat/diff.emu

```diff
@@ -1018,7 +1018,10 @@
           1. Else,
             1. Let _str_ be ! Number::toString(_x_).
         1. If the grammar cannot interpret _str_ as an expansion of |StringNumericLiteral|, return *NaN*.
-        1. Let _mv_ be the MV, a mathematical value, of ? ToNumber(_str_), as described in <emu-xref href="#sec-runtime-semantics-mv-s"></emu-xref>.
+        1. Let _text_ be ! StringToCodePoints(_str_).
+        1. Let _literal_ be ParseText(_text_, |StringNumericLiteral|).
+        1. If _literal_ is a List of errors, let _mv_ be *NaN*.
+        1. Else, let _mv_ be the StringIntlMV of _literal_.
```

Comment:
Where is "StringIntlMV" defined?

Reply:
I was wondering if was if a StringNumericValue "variation" was needed by copying over the whole grammar and add as with different naming in 262

Reply:
No, as mentioned [here](https://github.com/tc39/proposal-intl-numberformat-v3/pull/72#issuecomment-965835411), you do not need to copy the grammar itself, just StringNumericValue.

That is, the `let _mv_ be the StringIntlMV of _literal_.` bit is right, you just need to actually define what StringIntlMV is over the |StringNumericLiteral| grammar.